### PR TITLE
Use standard html instead of quirks html

### PR DESCRIPTION
**Pull request by greggman**

### CODE REVIEW

File: standalone/index.html [hunk 1]
```diff
@@ -16,6 +17,7 @@
     <meta http-equiv="origin-trial" content="AmV1vLgjOQ01SlGnVhpoKXy7gLW+K/plXHwHKnYn4S4US98WaSesKBI+XSUMo95unQARyMGDvW70KsfyeYblZQ0AAABQeyJvcmlnaW4iOiJodHRwczovL2dwdXdlYi5naXRodWIuaW86NDQzIiwiZmVhdHVyZSI6IldlYkdQVSIsImV4cGlyeSI6MTY2MzcxODM5OX0=">
     <link rel="stylesheet" href="third_party/normalize.min.css" />
     <script src="third_party/jquery/jquery-3.3.1.min.js"></script>
+    <script src="http://localhost:8081/dist/0.x/webgpu-debug-helper.js"></script>
```
Comment (kainino0x) [inline standalone/index.html:20]: Unrelated?

Comment (greggman) [reply standalone/index.html:20]: doh! I thought I removed that before commiting

Comment (greggman) [reply standalone/index.html]: fixed

**Review by kainino0x — APPROVED** [review]
